Libraries/LibCore/SocketAddress.h

```diff
@@ -82,12 +82,15 @@ class SocketAddress {
         }
     }
 
-    sockaddr_un to_sockaddr_un() const
+    Optional<sockaddr_un> to_sockaddr_un() const
     {
         ASSERT(type() == Type::Local);
         sockaddr_un address;
         address.sun_family = AF_LOCAL;
-        RELEASE_ASSERT(m_local_address.length() < (int)sizeof(address.sun_path));
+        if (m_local_address.length() >= sizeof(address.sun_path)) {
+            errno = E2BIG;
```

Comment:
LibCore does not communicate errors through `errno`

Reply:
Then I'll remove it.